mail/mlmmj/files/include-xlocale_h.diff

```diff
@@ -0,0 +1,10 @@
+--- src/strgen.c.orig	2023-07-03 21:18:19.000000000 +0900
++++ src/strgen.c	2024-06-21 23:23:14.000000000 +0900
+@@ -32,6 +32,7 @@
+ #include <ctype.h>
+ #include <errno.h>
+ #include <locale.h>
++#include <xlocale.h>
```

Comment:
An upstream bug report or pull request URL at the top of this patchfile would be appreciated.

Reply:
Upstream has fixed this issue, and published as mlmmj 1.4.7. I updated the Portfile.